diff -uNr OpenCASCADE6.3.0-old/ros/adm/make/Makefile.am OpenCASCADE6.3.0/ros/adm/make/Makefile.am
--- OpenCASCADE6.3.0-old/ros/adm/make/Makefile.am	2008-10-02 23:55:31.000000000 +0200
+++ OpenCASCADE6.3.0/ros/adm/make/Makefile.am	2008-10-02 23:56:45.000000000 +0200
@@ -19,7 +19,7 @@
   ModelingAlgorithms_DIRS = 
 endif
 
-Visualization_PKGS = TKService TKV2d TKV3d TKOpenGl TKMeshVS TKNIS 
+Visualization_PKGS = TKService TKV2d TKV3d TKOpenGl TKMeshVS TKNIS TKVRML 
 
 if ENABLE_VIS
   Visualization_DIRS = $(Visualization_PKGS)
@@ -35,7 +35,7 @@
   ApplicationFramework_DIRS = 
 endif
 
-DataExchange_PKGS = TKVRML TKXCAF TKSTL TKXSBase TKXCAFSchema TKXmlXCAF TKBinXCAF TKSTEPBase TKIGES TKSTEPAttr TKSTEP209 TKXDEIGES TKSTEP TKXDESTEP XCAFPlugin XmlXCAFPlugin BinXCAFPlugin 
+DataExchange_PKGS = TKXCAF TKSTL TKXSBase TKXCAFSchema TKXmlXCAF TKBinXCAF TKSTEPBase TKIGES TKSTEPAttr TKSTEP209 TKXDEIGES TKSTEP TKXDESTEP XCAFPlugin XmlXCAFPlugin BinXCAFPlugin 
 
 if ENABLE_DE
   DataExchange_DIRS = $(DataExchange_PKGS)
