diff -uNr OpenCASCADE6.3.0-old/ros/src/Aspect/Aspect_Driver.cxx OpenCASCADE6.3.0/ros/src/Aspect/Aspect_Driver.cxx
--- OpenCASCADE6.3.0-old/ros/src/Aspect/Aspect_Driver.cxx	2001-11-15 18:01:55.000000000 +0100
+++ OpenCASCADE6.3.0/ros/src/Aspect/Aspect_Driver.cxx	2008-10-02 23:38:50.000000000 +0200
@@ -35,14 +35,12 @@
   fontdir =  getenv("CSF_MDTVFontDirectory" );
   if ( !fontdir) {
     casroot  = getenv("CASROOT");
-    if ( casroot ) {
-      TCollection_AsciiString CasRootString (casroot);
-      CasRootString += "/src/FontMFT" ; 
-      CSF_MDTVFontDirectory = OSD_Environment(CasRootString.ToCString());
-    } else {
-      cout << " CASROOT or CSF_MDTVFontDirectory  are mandatory to use this fonctionnality " << endl;
-      Standard_Failure::Raise ( "CSF_MDTVTexturesDirectory and CASROOT not setted " );
+    if ( !casroot ) {
+      casroot = "/usr/share/OpenCASCADE";
     }
+    TCollection_AsciiString CasRootString (casroot);
+    CasRootString += "/src/FontMFT" ; 
+    CSF_MDTVFontDirectory = OSD_Environment(CasRootString.ToCString());
   } else {
     CSF_MDTVFontDirectory = OSD_Environment("CSF_MDTVFontDirectory");
   }
diff -uNr OpenCASCADE6.3.0-old/ros/src/DPrsStd/DPrsStd.cxx OpenCASCADE6.3.0/ros/src/DPrsStd/DPrsStd.cxx
--- OpenCASCADE6.3.0-old/ros/src/DPrsStd/DPrsStd.cxx	2005-07-08 15:51:41.000000000 +0200
+++ OpenCASCADE6.3.0/ros/src/DPrsStd/DPrsStd.cxx	2008-10-02 23:38:50.000000000 +0200
@@ -59,11 +59,12 @@
   Standard_Boolean hasPlugin = Standard_False; 
   TCollection_AsciiString PluginName ;
   if ( !plugin ) { 
-    if ( casroot ) {
-      PluginName = TCollection_AsciiString (casroot);
-      PluginName+="/src/StdResource" ;
-      hasPlugin = Standard_True ;
+    if ( !casroot ) {
+      casroot = "/usr/share/OpenCASCADE";
     }
+    PluginName = TCollection_AsciiString (casroot);
+    PluginName+="/src/StdResource" ;
+    hasPlugin = Standard_True ;
   } else {
     PluginName = TCollection_AsciiString (plugin);
     hasPlugin = Standard_True ;
diff -uNr OpenCASCADE6.3.0-old/ros/src/Draw/Draw.cxx OpenCASCADE6.3.0/ros/src/Draw/Draw.cxx
--- OpenCASCADE6.3.0-old/ros/src/Draw/Draw.cxx	2008-07-25 16:03:00.000000000 +0200
+++ OpenCASCADE6.3.0/ros/src/Draw/Draw.cxx	2008-10-02 23:38:50.000000000 +0200
@@ -253,8 +253,7 @@
 #ifdef WNT
 	ReadInitFile("ddefault");
 #else
-	cout << " the CASROOT variable is mandatory to Run OpenCascade "<<endl;
-	cout << "No default file" << endl;
+	ReadInitFile("/usr/share/OpenCASCADE/src/DrawResources/DrawDefault");
 #endif
       } else {
      
diff -uNr OpenCASCADE6.3.0-old/ros/src/Draw/Draw_PloadCommands.cxx OpenCASCADE6.3.0/ros/src/Draw/Draw_PloadCommands.cxx
--- OpenCASCADE6.3.0-old/ros/src/Draw/Draw_PloadCommands.cxx	2008-08-15 15:57:21.000000000 +0200
+++ OpenCASCADE6.3.0/ros/src/Draw/Draw_PloadCommands.cxx	2008-10-02 23:38:50.000000000 +0200
@@ -54,13 +54,11 @@
     // now try by CASROOT
     aPluginDir = getenv("CASROOT");
 
-    if ( !aPluginDir.IsEmpty() ) {
-      aPluginDir +="/src/DrawResources" ;
-      aToSetCSFVariable = Standard_True; //CSF variable to be set later
-    } else {
-      aResult = aDirFound = Standard_False;
-      cout << FAILSTR "Neither " << aCSFVariable.ToCString() << ", nor CASROOT variables have been set" << endl;
+    if ( aPluginDir.IsEmpty() ) {
+      aPluginDir = "/usr/share/OpenCASCADE";
     }
+    aPluginDir +="/src/DrawResources" ;
+    aToSetCSFVariable = Standard_True; //CSF variable to be set later
   }
   
   if (aDirFound) {
diff -uNr OpenCASCADE6.3.0-old/ros/src/DrawResources/DrawDefault OpenCASCADE6.3.0/ros/src/DrawResources/DrawDefault
--- OpenCASCADE6.3.0-old/ros/src/DrawResources/DrawDefault	2003-10-03 19:01:22.000000000 +0200
+++ OpenCASCADE6.3.0/ros/src/DrawResources/DrawDefault	2008-10-02 23:38:50.000000000 +0200
@@ -8,11 +8,10 @@
 if { [info exists env(DRAWHOME) ] } { 
   set dir $env(DRAWHOME)  
 } else {
-    if { [info exists env(CASROOT) ] } {
-  set dir [file join $env(CASROOT) src DrawResources]
-    } else {
-  puts " CASROOT is Mandatory to Run OpenCascade"
+    if { ![info exists env(CASROOT) ] } {
+      set env(CASROOT) "/usr/share/OpenCASCADE"
     }
+  set dir [file join $env(CASROOT) src DrawResources]
 }
 
 # san - 02/08/2002 - In Tcl8.3 it is no longer necessary to load Tix explicitly on WNT
diff -uNr OpenCASCADE6.3.0-old/ros/src/Graphic3d/Graphic3d_GraphicDevice.cxx OpenCASCADE6.3.0/ros/src/Graphic3d/Graphic3d_GraphicDevice.cxx
--- OpenCASCADE6.3.0-old/ros/src/Graphic3d/Graphic3d_GraphicDevice.cxx	2008-07-04 17:36:54.000000000 +0200
+++ OpenCASCADE6.3.0/ros/src/Graphic3d/Graphic3d_GraphicDevice.cxx	2008-10-02 23:38:50.000000000 +0200
@@ -204,29 +204,8 @@
 
   if (! BAD(glso)) {
     glshr = getenv("CSF_GraphicShr");
-  } else if (! BAD(casroot)) {
-    glshr = (char *) malloc (128);
-    /* Pour faire plaisir a Purify */
-    glshr[0] = '\0';
-    strcat(glshr, casroot);
-    struct utsname info; 
-    uname (&info);
-    strcat(glshr,"/");
-    strcat(glshr,info.sysname);   
-#if defined(__hpux) || defined(HPUX)
-    strcat (glshr, "/lib/libTKOpenGl.sl");
-#elif defined(WNT)
-    strcat (glshr, "/TKOpenGl.dll");
-#elif defined(__APPLE__)
-    strcat (glshr, "/lib/libTKOpenGl.dylib");
-#else
-    strcat (glshr, "/lib/libTKOpenGl.so");
-#endif
-//    printf("You have not defined CSF_GraphicShr, defaulting to %s",glshr);
   } else {
-    aShr = NULL;
-    printf("You have not defined CSF_GraphicShr or CASROOT, aborting...");
-    return Standard_False;
+    glshr = "/usr/lib/libTKOpenGl.so";
   }
  
  aShr = glshr;
diff -uNr OpenCASCADE6.3.0-old/ros/src/Graphic3d/Graphic3d_Texture1D.cxx OpenCASCADE6.3.0/ros/src/Graphic3d/Graphic3d_Texture1D.cxx
--- OpenCASCADE6.3.0-old/ros/src/Graphic3d/Graphic3d_Texture1D.cxx	2008-07-04 17:39:03.000000000 +0200
+++ OpenCASCADE6.3.0/ros/src/Graphic3d/Graphic3d_Texture1D.cxx	2008-10-02 23:38:50.000000000 +0200
@@ -32,11 +32,12 @@
     Standard_Boolean HasDefinition = Standard_False ;
     if ( !envir ) { 
       casroot  = getenv("CASROOT");
-      if ( casroot ) {
-	VarName = TCollection_AsciiString  (casroot);
-	VarName += "/src/Textures" ;
-	HasDefinition = Standard_True ;
+      if ( !casroot ) {
+	casroot = "/usr/share/OpenCASCADE";
       }
+      VarName = TCollection_AsciiString  (casroot);
+      VarName += "/src/Textures" ;
+      HasDefinition = Standard_True ;
     } else {
       VarName = TCollection_AsciiString  (envir);
       HasDefinition = Standard_True ;
diff -uNr OpenCASCADE6.3.0-old/ros/src/Graphic3d/Graphic3d_Texture2D.cxx OpenCASCADE6.3.0/ros/src/Graphic3d/Graphic3d_Texture2D.cxx
--- OpenCASCADE6.3.0-old/ros/src/Graphic3d/Graphic3d_Texture2D.cxx	2008-07-04 17:39:03.000000000 +0200
+++ OpenCASCADE6.3.0/ros/src/Graphic3d/Graphic3d_Texture2D.cxx	2008-10-02 23:38:50.000000000 +0200
@@ -28,11 +28,12 @@
     Standard_Boolean HasDefinition = Standard_False ;
     if ( !envir ) { 
       casroot  = getenv("CASROOT");
-      if ( casroot ) {
-	VarName = TCollection_AsciiString  (casroot);
-	VarName += "/src/Textures" ;
-	HasDefinition = Standard_True ;
+      if ( !casroot ) {
+	casroot = "/usr/share/OpenCASCADE";
       }
+      VarName = TCollection_AsciiString  (casroot);
+      VarName += "/src/Textures" ;
+      HasDefinition = Standard_True ;
     } else {
       VarName = TCollection_AsciiString  (envir);
       HasDefinition = Standard_True ;
diff -uNr OpenCASCADE6.3.0-old/ros/src/Graphic3d/Graphic3d_TextureEnv.cxx OpenCASCADE6.3.0/ros/src/Graphic3d/Graphic3d_TextureEnv.cxx
--- OpenCASCADE6.3.0-old/ros/src/Graphic3d/Graphic3d_TextureEnv.cxx	2008-07-04 17:39:03.000000000 +0200
+++ OpenCASCADE6.3.0/ros/src/Graphic3d/Graphic3d_TextureEnv.cxx	2008-10-02 23:38:50.000000000 +0200
@@ -29,11 +29,12 @@
     Standard_Boolean HasDefinition = Standard_False ;
     if ( !envir ) { 
       casroot  = getenv("CASROOT");
-      if ( casroot ) {
-	VarName = TCollection_AsciiString  (casroot);
-	VarName += "/src/Textures" ;
-	HasDefinition = Standard_True ;
+      if ( !casroot ) {
+	casroot = "/usr/share/OpenCASCADE";
       }
+      VarName = TCollection_AsciiString  (casroot);
+      VarName += "/src/Textures" ;
+      HasDefinition = Standard_True ;
     } else {
       VarName = TCollection_AsciiString  (envir);
       HasDefinition = Standard_True ;
diff -uNr OpenCASCADE6.3.0-old/ros/src/UnitsAPI/UnitsAPI.cxx OpenCASCADE6.3.0/ros/src/UnitsAPI/UnitsAPI.cxx
--- OpenCASCADE6.3.0-old/ros/src/UnitsAPI/UnitsAPI.cxx	2006-08-04 19:03:13.000000000 +0200
+++ OpenCASCADE6.3.0/ros/src/UnitsAPI/UnitsAPI.cxx	2008-10-02 23:38:50.000000000 +0200
@@ -35,13 +35,11 @@
     else {
       OSD_Environment CasRootEnv("CASROOT");
       TCollection_AsciiString CasRootString(CasRootEnv.Value());
-      if (CasRootString.Length() > 0 )  {
-	CasRootString += "/src/UnitsAPI/Lexi_Expr.dat" ;
-	Units::LexiconFile(CasRootString.ToCString());
-      }
-      else {
-	Standard_NoSuchObject::Raise("environment variable CSF_UnitsLexicon undefined");
+      if (CasRootString.Length() == 0 )  {
+        CasRootString = "/usr/share/OpenCASCADE";
       }
+      CasRootString += "/src/UnitsAPI/Lexi_Expr.dat" ;
+      Units::LexiconFile(CasRootString.ToCString());
     }
     TCollection_AsciiString sunitsfile(env2.Value());
     if( sunitsfile.Length() > 0 )
@@ -49,13 +47,11 @@
     else {
       OSD_Environment CasRootEnv("CASROOT");
       TCollection_AsciiString CasRootString(CasRootEnv.Value());
-      if (CasRootString.Length() > 0 ) {
-	CasRootString += "/src/UnitsAPI/Units.dat";
-	Units::UnitsFile(CasRootString.ToCString());
-      }
-      else {
-	Standard_NoSuchObject::Raise("environment variable CSF_UnitsDefinition undefined");
+      if (CasRootString.Length() == 0 ) {
+        CasRootString = "/usr/share/OpenCASCADE";
       }
+      CasRootString += "/src/UnitsAPI/Units.dat";
+      Units::UnitsFile(CasRootString.ToCString());
     }  
     switch (aSystemUnits) {
       case UnitsAPI_DEFAULT :
diff -uNr OpenCASCADE6.3.0-old/ros/src/ViewerTest/ViewerTest.cxx OpenCASCADE6.3.0/ros/src/ViewerTest/ViewerTest.cxx
--- OpenCASCADE6.3.0-old/ros/src/ViewerTest/ViewerTest.cxx	2008-08-15 15:57:34.000000000 +0200
+++ OpenCASCADE6.3.0/ros/src/ViewerTest/ViewerTest.cxx	2008-10-02 23:38:50.000000000 +0200
@@ -1566,11 +1566,12 @@
     Standard_Boolean HasDefinition = Standard_False ;
     if ( !envir ) { 
       casroot  = getenv("CASROOT");
-      if ( casroot ) {
-	VarName = TCollection_AsciiString  (casroot);
-	VarName += "/src/Textures" ;
-	HasDefinition = Standard_True ;
+      if ( !casroot ) {
+	casroot = "/usr/share/OpenCASCADE";
       }
+      VarName = TCollection_AsciiString  (casroot);
+      VarName += "/src/Textures" ;
+      HasDefinition = Standard_True ;
     } else {
       VarName = TCollection_AsciiString  (envir);
       HasDefinition = Standard_True ;
diff -uNr OpenCASCADE6.3.0-old/ros/src/XmlLDrivers/XmlLDrivers_DocumentStorageDriver.cxx OpenCASCADE6.3.0/ros/src/XmlLDrivers/XmlLDrivers_DocumentStorageDriver.cxx
--- OpenCASCADE6.3.0-old/ros/src/XmlLDrivers/XmlLDrivers_DocumentStorageDriver.cxx	2008-07-04 17:39:02.000000000 +0200
+++ OpenCASCADE6.3.0/ros/src/XmlLDrivers/XmlLDrivers_DocumentStorageDriver.cxx	2008-10-02 23:38:50.000000000 +0200
@@ -183,18 +183,11 @@
   if (aResourceDir.IsEmpty()) {
     // now try by CASROOT
     aResourceDir = getenv (aCSFVariable[1]);
-    if ( !aResourceDir.IsEmpty() ) {
-      aResourceDir += "/src/XmlOcafResource" ;
-      aToSetCSFVariable = Standard_True; //CSF variable to be set later
+    if ( aResourceDir.IsEmpty() ) {
+      aResourceDir = "/usr/share/OpenCASCADE";
     }
-#ifdef DEB
-    else {
-      TCollection_ExtendedString aWarn = FAILSTR "Neither ";
-      aWarn = (aWarn + aCSFVariable[0] + ", nor " + aCSFVariable[1]
-               + " variables have been set");
-      aMessageDriver->Write (aWarn.ToExtString());
-    }
-#endif
+    aResourceDir += "/src/XmlOcafResource" ;
+    aToSetCSFVariable = Standard_True; //CSF variable to be set later
   }
   if (!aResourceDir.IsEmpty()) {
     TCollection_AsciiString aResourceFileName =  aResourceDir + "/XmlOcaf.xsd";
