diff -uNr OpenCASCADE6.3.0-old/ros/adm/make/TKernel/Makefile.am OpenCASCADE6.3.0/ros/adm/make/TKernel/Makefile.am
--- OpenCASCADE6.3.0-old/ros/adm/make/TKernel/Makefile.am	2008-10-02 23:40:38.000000000 +0200
+++ OpenCASCADE6.3.0/ros/adm/make/TKernel/Makefile.am	2008-10-02 23:43:05.000000000 +0200
@@ -55,7 +55,7 @@
 lib_LTLIBRARIES=libTKernel.la
 
 libTKernel_la_LIBADD = \
-$(STLPort_LIB) $(CSF_SOCKETLibs_LIB) $(CSF_FPELibs_LIB) $(CSF_ThreadLibs_LIB) $(CSF_advapi32_LIB) $(CSF_gdi32_LIB) $(CSF_user32_LIB) $(CSF_kernel32_LIB) 
+$(STLPort_LIB) $(CSF_SOCKETLibs_LIB) $(CSF_FPELibs_LIB) $(CSF_ThreadLibs_LIB) $(CSF_advapi32_LIB) $(CSF_gdi32_LIB) $(CSF_user32_LIB) $(CSF_kernel32_LIB) -ldl
 
 libTKernel_la_SOURCES = \
 @top_srcdir@/drv/Dico/Dico_DictionaryOfInteger_0.cxx \
