diff -uNr OpenCASCADE6.3.0-old/ros/inc/MFT_FileRecord.hxx OpenCASCADE6.3.0/ros/inc/MFT_FileRecord.hxx
--- OpenCASCADE6.3.0-old/ros/inc/MFT_FileRecord.hxx	2008-10-02 23:41:46.000000000 +0200
+++ OpenCASCADE6.3.0/ros/inc/MFT_FileRecord.hxx	2008-10-02 23:49:44.000000000 +0200
@@ -19,6 +19,7 @@
 */
 #endif	//WNT
 
+#if 0
 #if defined ( __hpux ) || defined ( HPUX )
 #include <sys/mman.h>
 #define MAP_FAILED -1
@@ -55,6 +56,7 @@
 #define MUNMAP(mmap_addr,mmap_size) \
         munmap((caddr_t) mmap_addr,(size_t) mmap_size)
 #endif 	// __alpha
+#endif
 
 #ifndef MMAP
 // #error  *** MMAP is not used on your system by MFT fonts ***
