--- occt-cec1ecd/src/IVtkDraw/IVtkDraw_Interactor.cxx.orig	2023-07-21 17:36:51.000000000 +0200
+++ occt-cec1ecd/src/IVtkDraw/IVtkDraw_Interactor.cxx	2023-09-06 19:55:42.845602803 +0200
@@ -19,6 +19,8 @@
 #include <vtkWin32RenderWindowInteractor.h>
 #include <vtkWin32OpenGLRenderWindow.h>
 #else
+#include <vtkXRenderWindowInteractor.h>
+#include <vtkXOpenGLRenderWindow.h>
 #include <GL/glx.h>
 
 // Preventing naming collisions between
@@ -36,8 +38,6 @@
 #undef Success
 #endif
 
-#include <vtkXRenderWindowInteractor.h>
-#include <vtkXOpenGLRenderWindow.h>
 #endif
 #include <vtkActor.h>
 #include <vtkActorCollection.h>
