diff -uNr OpenCASCADE6.3.0-old/ros/Makefile.am OpenCASCADE6.3.0/ros/Makefile.am
--- OpenCASCADE6.3.0-old/ros/Makefile.am	2008-10-02 23:57:41.000000000 +0200
+++ OpenCASCADE6.3.0/ros/Makefile.am	2008-10-02 23:59:13.000000000 +0200
@@ -82,8 +82,10 @@
 WOK_PKGS = WOKBuilderDef WOKEntityDef WOKStepsDef WOKTclLib WOKsite 
 if ENABLE_WOK
   WOK_DIRS = $(WOK_PKGS)
+  WOK_EXTRA_FILES = CPPClient/CPPClient_General.edl CPPClient/CPPClient_Template.edl CPPExt/CPPExt_Standard.edl CPPExt/CPPExt_TemplateCSFDB.edl CPPExt/CPPExt_Template.edl CPPExt/CPPExt_TemplateOBJS.edl CPPExt/CPPExt_TemplateOBJY.edl CPPJini/CPPJini_General.edl CPPJini/CPPJini_Template.edl CSFDBSchema/CSFDBSchema_Template.edl CPPIntExt/Engine_Template.edl WOKTclTools/ENV.edl WOKOrbix/*.edl WOKOrbix/*.tcl CPPIntExt/Interface_Template.edl WOKOBJS/*.edl TCPPExt/TCPPExt_MethodTemplate.edl WOKUtils/EDL.edl WOKDeliv/WOKDeliv_FRONTALSCRIPT.edl WOKDeliv/WOKDeliv_LDSCRIPT.edl WOKDeliv/WOKDeliv_DelivExecSource.tcl WOKStep/*.tcl
 else
   WOK_DIRS = 
+  WOK_EXTRA_FILES =
 endif
 
 
@@ -215,9 +217,21 @@
 	if [ "$$pd" != "$$sd" ]; then \
 		inst_dirs="$(WOK_DIRS)"; \
 		if test "x$(WOK_DIRS)" != "x"; then \
+			$(INSTALL) -d $(DESTDIR)$(prefix)/wok/lib; \
+			$(INSTALL) -d $(DESTDIR)$(prefix)/wok/site; \
 			for d in $$inst_dirs; do \
-				$(INSTALL) -d $(DESTDIR)$(prefix)/src/$$d; \
-				cp -fr$(CP_$(platform)) $(top_srcdir)/src/$$d $(DESTDIR)$(prefix)/src; \
+				if test "$$d" = WOKsite; then \
+					cp -fr$(CP_$(platform)) $(top_srcdir)/src/$$d/* $(DESTDIR)$(prefix)/wok/site; \
+				else \
+					cp -fr$(CP_$(platform)) $(top_srcdir)/src/$$d/* $(DESTDIR)$(prefix)/wok/lib; \
+				fi; \
+			done; \
+		fi; \
+		inst_files="$(WOK_EXTRA_FILES)"; \
+		if test "x$(WOK_EXTRA_FILES)" != "x"; then \
+			$(INSTALL) -d $(DESTDIR)$(prefix)/wok/lib; \
+			for f in $$inst_files; do \
+				cp -fr$(CP_$(platform)) $(top_srcdir)/src/$$f $(DESTDIR)$(prefix)/wok/lib; \
 			done; \
 		fi; \
 	fi
