--- OCCT-7_9_3/tools/TInspectorEXE/TInspectorEXE.cxx.orig	2025-12-06 00:55:46.000000000 +0100
+++ OCCT-7_9_3/tools/TInspectorEXE/TInspectorEXE.cxx	2026-09-20 19:25:04.054263945 +0200
@@ -42,6 +42,8 @@ TCollection_AsciiString fileNameInDataDi
   OSD_Environment anEnvironment(theEnvironmentDir);
 
   TCollection_AsciiString aFileName = anEnvironment.Value();
+  if (aFileName.IsEmpty())
+    aFileName = "/usr/share/opencascade/data";
   aFileName += TCollection_AsciiString("/") + theName;
 
   return aFileName;
