diff -uNr old-OpenCASCADE6.3.0/ros/src/ExprIntrp/ExprIntrp.yacc OpenCASCADE6.3.0/ros/src/ExprIntrp/ExprIntrp.yacc
--- old-OpenCASCADE6.3.0/ros/src/ExprIntrp/ExprIntrp.yacc	2001-09-21 09:50:48.000000000 +0200
+++ OpenCASCADE6.3.0/ros/src/ExprIntrp/ExprIntrp.yacc	2008-10-10 23:36:36.000000000 +0200
@@ -42,6 +42,10 @@
 extern void ExprIntrp_VariableIdentifier();
 extern void ExprIntrp_Productor();
 extern void ExprIntrp_EndOfEqual();
+
+int yylex(void);
+void yyerror(char const *s);
+
 %}
 
 %token SUMOP MINUSOP DIVIDEOP EXPOP MULTOP PARENTHESIS BRACKET ENDPARENTHESIS ENDBRACKET VALUE IDENTIFIER COMMA DIFFERENTIAL DERIVATE DERIVKEY ASSIGNOP DEASSIGNKEY EQUALOP RELSEPARATOR CONSTKEY SUMKEY PRODKEY
diff -uNr old-OpenCASCADE6.3.0/ros/src/OpenGl/OpenGl_telem_util.h OpenCASCADE6.3.0/ros/src/OpenGl/OpenGl_telem_util.h
--- old-OpenCASCADE6.3.0/ros/src/OpenGl/OpenGl_telem_util.h	2008-06-27 17:50:06.000000000 +0200
+++ OpenCASCADE6.3.0/ros/src/OpenGl/OpenGl_telem_util.h	2008-10-10 23:39:33.000000000 +0200
@@ -155,6 +155,9 @@
                                      Tfloat *xr, Tfloat *yr);
 extern  TStatus TelUnProjectionRaster( Tint ws, Tint xr, Tint yr,
                                      Tfloat *x, Tfloat *y, Tfloat *z);
+extern TStatus TelUnProjectionRasterWithRay( Tint ws, Tint xr, Tint yr,
+ Tfloat *x, Tfloat *y, Tfloat *z,
+ Tfloat *dx, Tfloat *dy, Tfloat *dz)
 extern  Tint  TelBackBufferRestored(void);
 extern  void  TelSetBackBufferRestored( Tint );
 extern  void  TelEnable( Tint );
diff -uNr old-OpenCASCADE6.3.0/ros/src/OpenGl/OpenGl_tXfm.c OpenCASCADE6.3.0/ros/src/OpenGl/OpenGl_tXfm.c
--- old-OpenCASCADE6.3.0/ros/src/OpenGl/OpenGl_tXfm.c	2008-08-15 15:54:07.000000000 +0200
+++ OpenCASCADE6.3.0/ros/src/OpenGl/OpenGl_tXfm.c	2008-10-10 23:38:25.000000000 +0200
@@ -64,6 +64,7 @@
 #include <X11/keysym.h>
 
 #include <GL/glx.h>
+#include <GL/glu.h>
 
 #include <OpenGl_tgl_all.h>
 #include <OpenGl_tXfm.h>
@@ -79,6 +80,7 @@
 #include <OpenGl_cmn_varargs.h>
 #include <OpenGl_tsm.h>
 #include <OpenGl_tgl_funcs.h>
+#include <OpenGl_telem_attri.h>
 /*----------------------------------------------------------------------*/
 /*
  * Constantes
