diff -uNr OpenCASCADE6.3.0-old/ros/adm/make/WOKLibs/Makefile.am OpenCASCADE6.3.0/ros/adm/make/WOKLibs/Makefile.am
--- OpenCASCADE6.3.0-old/ros/adm/make/WOKLibs/Makefile.am	2008-10-02 23:56:22.000000000 +0200
+++ OpenCASCADE6.3.0/ros/adm/make/WOKLibs/Makefile.am	2008-10-02 23:57:31.000000000 +0200
@@ -9,7 +9,7 @@
 -I@top_srcdir@/src/WOKLibs
 
  
-lib_LTLIBRARIES= libwokcmd.la libwoktoolscmd.la libwokutilscmd.la libwoksteps.la libwokobjssteps.la libwokdfltsteps.la libwokdeliverysteps.la libwokorbixsteps.la libmscmd.la
+pkglib_LTLIBRARIES= libwokcmd.la libwoktoolscmd.la libwokutilscmd.la libwoksteps.la libwokobjssteps.la libwokdfltsteps.la libwokdeliverysteps.la libwokorbixsteps.la libmscmd.la
  libwokcmd_la_SOURCES = @top_srcdir@/src/WOKLibs/wokcmd.cxx 
  libwoktoolscmd_la_SOURCES = @top_srcdir@/src/WOKLibs/woktoolscmd.cxx 
  libwokutilscmd_la_SOURCES = @top_srcdir@/src/WOKLibs/wokutilscmd.cxx 
@@ -64,12 +64,13 @@
 ../TKAdvTools/libTKAdvTools.la \
 ../TKernel/libTKernel.la \
  $(STLPort_LIB) $(CSF_TclLibs_LIB) $(CSF_TclTkLibs_LIB) $(CSF_SOCKETLibs_LIB) $(CSF_FPELibs_LIB) $(CSF_ThreadLibs_LIB) $(CSF_advapi32_LIB) $(CSF_gdi32_LIB) $(CSF_user32_LIB) $(CSF_kernel32_LIB)  
- libwokcmd_la_LDFLAGS = -release 6.3.0
- libwoktoolscmd_la_LDFLAGS = -release 6.3.0
- libwokutilscmd_la_LDFLAGS = -release 6.3.0
- libwoksteps_la_LDFLAGS = -release 6.3.0
- libwokobjssteps_la_LDFLAGS = -release 6.3.0
- libwokdfltsteps_la_LDFLAGS = -release 6.3.0
- libwokdeliverysteps_la_LDFLAGS = -release 6.3.0
- libwokorbixsteps_la_LDFLAGS = -release 6.3.0
- libmscmd_la_LDFLAGS = -release 6.3.0
+
+libwokcmd_la_LDFLAGS = -avoid-version
+libwoktoolscmd_la_LDFLAGS = -avoid-version
+libwokutilscmd_la_LDFLAGS = -avoid-version
+libwoksteps_la_LDFLAGS = -avoid-version
+libwokobjssteps_la_LDFLAGS = -avoid-version
+libwokdfltsteps_la_LDFLAGS = -avoid-version
+libwokdeliverysteps_la_LDFLAGS = -avoid-version
+libwokorbixsteps_la_LDFLAGS = -avoid-version
+libmscmd_la_LDFLAGS = -avoid-version
