diff -uNr OpenCASCADE6.3.0-old/ros/inc/InterfaceGraphic_X11.hxx OpenCASCADE6.3.0/ros/inc/InterfaceGraphic_X11.hxx
--- OpenCASCADE6.3.0-old/ros/inc/InterfaceGraphic_X11.hxx	2008-10-02 23:49:55.000000000 +0200
+++ OpenCASCADE6.3.0/ros/inc/InterfaceGraphic_X11.hxx	2008-10-02 23:51:19.000000000 +0200
@@ -26,6 +26,7 @@
 #  include <X11/Xutil.h>
 #  include <X11/Xatom.h>
 
+#if 0
 #  if defined (sun) || defined (SUNOS) || defined (__alpha) || defined (DECOSF1) || defined (sgi) || defined (IRIX) || defined (__hpux)|| defined (HPUX) 
 #   ifndef icon_width
 #    include <X11/bitmaps/icon>
@@ -46,6 +47,7 @@
 #   endif
 
 #  endif /* ULTRIX */
+#endif
 #  define WINDOW Window
 #  ifndef EXPORT
 #   define EXPORT
diff -uNr OpenCASCADE6.3.0-old/ros/src/InterfaceGraphic/InterfaceGraphic_X11.hxx OpenCASCADE6.3.0/ros/src/InterfaceGraphic/InterfaceGraphic_X11.hxx
--- OpenCASCADE6.3.0-old/ros/src/InterfaceGraphic/InterfaceGraphic_X11.hxx	2008-10-02 23:50:04.000000000 +0200
+++ OpenCASCADE6.3.0/ros/src/InterfaceGraphic/InterfaceGraphic_X11.hxx	2008-10-02 23:51:19.000000000 +0200
@@ -26,6 +26,7 @@
 #  include <X11/Xutil.h>
 #  include <X11/Xatom.h>
 
+#if 0
 #  if defined (sun) || defined (SUNOS) || defined (__alpha) || defined (DECOSF1) || defined (sgi) || defined (IRIX) || defined (__hpux)|| defined (HPUX) 
 #   ifndef icon_width
 #    include <X11/bitmaps/icon>
@@ -46,6 +47,7 @@
 #   endif
 
 #  endif /* ULTRIX */
+#endif
 #  define WINDOW Window
 #  ifndef EXPORT
 #   define EXPORT
