diff -uNr OpenCASCADE6.3.0-old/ros/adm/make/WOKLibs/Makefile.am OpenCASCADE6.3.0/ros/adm/make/WOKLibs/Makefile.am
--- OpenCASCADE6.3.0-old/ros/adm/make/WOKLibs/Makefile.am	2008-10-02 23:56:22.000000000 +0200
+++ OpenCASCADE6.3.0/ros/adm/make/WOKLibs/Makefile.am	2008-10-02 23:57:31.000000000 +0200
@@ -9,7 +9,7 @@
 -I@top_srcdir@/src/WOKLibs
 
  
-lib_LTLIBRARIES= libwokcmd.la libwoktoolscmd.la libwokutilscmd.la libwoksteps.la libwokobjssteps.la libwokdfltsteps.la libwokdeliverysteps.la libwokorbixsteps.la libmscmd.la
+pkglib_LTLIBRARIES= libwokcmd.la libwoktoolscmd.la libwokutilscmd.la libwoksteps.la libwokobjssteps.la libwokdfltsteps.la libwokdeliverysteps.la libwokorbixsteps.la libmscmd.la
  libwokcmd_la_SOURCES = @top_srcdir@/src/WOKLibs/wokcmd.cxx 
  libwoktoolscmd_la_SOURCES = @top_srcdir@/src/WOKLibs/woktoolscmd.cxx 
  libwokutilscmd_la_SOURCES = @top_srcdir@/src/WOKLibs/wokutilscmd.cxx 
